public/stylesheets/dark.css

```diff
@@ -443,6 +443,10 @@ body.dark #puzzle .please_vote {
   background: #103410;
   color: #74a962;
 }
+body.dark #team table.requests.for-team {
+  border-top: 1px solid #3d3d3d;
+  border-bottom: 1px solid #3d3d3d;
+}
```

Comment:
there's already a rule for this https://github.com/seanysean/lila/blob/0a2a5f4fd01c24edd8de1679a74b32e11f0f4b5e/public/stylesheets/dark.css#L184